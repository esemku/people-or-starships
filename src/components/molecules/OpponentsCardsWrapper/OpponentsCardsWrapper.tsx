import React, { ReactNode } from 'react';
import useStyles from './styles';

type OpponentsCardsWrapperProps = {
  children: ReactNode;
};

const OpponentsCardsWrapper: React.FC<OpponentsCardsWrapperProps> = ({
  children,
}: OpponentsCardsWrapperProps) => {
  const styles = useStyles();

  return (
    <div className={styles.root}>
      <div className={styles.cardsContainer}>{children}</div>
    </div>
  );
};

export default OpponentsCardsWrapper;
