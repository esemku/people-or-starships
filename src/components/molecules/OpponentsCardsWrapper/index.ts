export { default as OpponentsCardsWrapper } from './OpponentsCardsWrapper';
