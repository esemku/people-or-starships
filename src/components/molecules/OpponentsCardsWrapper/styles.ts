import { makeStyles } from '@mui/styles';

export default makeStyles({
  root: {
    marginTop: 40,
    marginBottom: 40,
  },
  cardsContainer: {
    display: 'flex',
    alignItems: 'center',
  },
});
