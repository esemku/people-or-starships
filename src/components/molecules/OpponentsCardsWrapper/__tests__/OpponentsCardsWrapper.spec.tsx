import React from 'react';
import { render } from '@testing-library/react';

import OpponentsCardsWrapper from '../OpponentsCardsWrapper';

describe('<OpponentsCardsWrapper />', () => {
  it('renders correctly', () => {
    const { container } = render(
      <OpponentsCardsWrapper>
        <div>children</div>
      </OpponentsCardsWrapper>,
    );

    expect(container).toMatchSnapshot();
  });
});
