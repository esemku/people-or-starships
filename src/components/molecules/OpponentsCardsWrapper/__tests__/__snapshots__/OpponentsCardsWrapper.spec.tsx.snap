// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OpponentsCardsWrapper /> renders correctly 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <div
      class="makeStyles-cardsContainer-2"
    >
      <div>
        children
      </div>
    </div>
  </div>
</div>
`;
